RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -469,26 +469,34 @@ class ProjectLoader {
 
         // Earliest check for valid input and output directories. If they are not accessible or writeable.
         // Start with the input directory
-        for (File _dir = analysis.getInputBaseDirectory(); _dir; _dir = _dir.parentFile) {
-            if (!FileSystemAccessProvider.instance.isReadable(_dir) || !FileSystemAccessProvider.instance.isExecutable(_dir)) {
-                errors << "The input directory was not readable at path ${_dir}."
-                break
-            }
-        }
-
-        for (File _dir = analysis.getOutputBaseDirectory(); _dir; _dir = _dir.parentFile) {
-            if (!FileSystemAccessProvider.instance.isReadable(_dir) || !FileSystemAccessProvider.instance.isExecutable(_dir))
-                if (!FileSystemAccessProvider.instance.isReadable(_dir)) {
-                    errors << "The output directory was not readable at path ${_dir}."
-                    break
-                }
-        }
+        errors += checkDirForReadabilityAndExecutability(analysis.getInputBaseDirectory())
```

Comment:
O.k. you don't want to discriminate between inner path elements (executability-only is fine) and terminal path elements (readable and executable is necessary (and writable for output dir). Fine.

Reply:
Don't know, if it is really necessary. Let's just see, how it behaves? We could also create a flag to ignore this.